service/collector/tiller_configured.go

```diff
@@ -0,0 +1,90 @@
+package collector
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	"github.com/prometheus/client_golang/prometheus"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	tillerConfiguredDesc *prometheus.Desc = prometheus.NewDesc(
+		prometheus.BuildFQName(Namespace, "", "tiller_configured"),
+		"Tiller is configured correctly.",
+		[]string{
+			namespaceLabel,
+		},
+		nil,
+	)
+)
+
+func (c *Collector) collectTillerConfigured(ctx context.Context, ch chan<- prometheus.Metric) {
```

Comment:
Even if it's a private method, I don't think it's easy to find out what is the purpose of this method.